morib/src/main/java/org/morib/server/api/allowGroupView/controller/AllowedGroupViewController.java

```diff
@@ -0,0 +1,26 @@
+package org.morib.server.api.allowGroupView.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.morib.server.api.allowGroupView.facade.AllowedGroupViewFacade;
+import org.morib.server.global.common.BaseResponse;
+import org.morib.server.global.message.SuccessMessage;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v2/allowServiceSet")
+public class AllowedGroupViewController {
+
+    private final AllowedGroupViewFacade allowedGroupViewFacade;
+
+    @DeleteMapping("/{groupId}")
+    public ResponseEntity<BaseResponse<?>> deleteAllowServiceSet(@PathVariable Long groupId){
+        allowedGroupViewFacade.deleteAllowServiceSet(groupId);
+        return ResponseEntity.ok(BaseResponse.of(SuccessMessage.SUCCESS));
```

Comment:
마지막 리턴문은 컨벤션?은 아니지만 기존 것과 동일하게 해주시면 좋을 것 같습니다!
```suggestion
return ApiResponseUtil.success(SuccessMessage.SUCCESS);
```

Reply:
아 맞네요! 그렇게 사용되는 부분이 있었네요! ㅎㅎ 다시 명심하고 고쳐보도록 하겠습니다!